micro-consumer/winston/winston.js

```diff
@@ -0,0 +1,35 @@
+var winston = require('winston');
```

Comment:
Consider using const for variables that don't need reassignment

Reply:
Thanks I miss that some times